dev/io.openliberty.pages.4.0.internal_fat/test-applications/TestEL.war/src/com/ibm/ws/jsp23/fat/testel/servlets/ResolutionVariablesPropertiesServlet.java

```diff
@@ -0,0 +1,168 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2023 IBM Corporation and others.
```

Comment:
Why a date range?

Reply:
It was copied from the `com.ibm.ws.jsp.2.3_fat` and then updated to work with Pages 4.0.